Pokemon Unity/Assets/Scripts2/Saving/SaveEvent.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using PokemonUnity.Saving.Location;
+
+namespace PokemonUnity.Saving
+{
+    [System.Serializable]
+    public class SaveEvent
+    {
+        public DateTime EventTime;
```

Comment:
Why are you storing event date time? Are you going to be using it again later? I can understand for a battle log and history. But if you're also logging everytime a player interacts with npc, i can't see much purpose for this.

Reply:
For now I'd like to keep it in (until I can find a better solution to save items).
The inspiration to use this for all the SaveEvents was because in Generation 4 you could find HM's in items that were lying on the ground: http://i55.tinypic.com/33f6z4m.png
Those will register as normal SaveEvents, but do indicate that the Player has found for example "Rock Climb" on Route 216. Would be nice to have in a log when you select your save file.